types/jsdom/base.d.ts

```diff
@@ -306,7 +306,6 @@ declare module "jsdom" {
         HTMLDataElement: typeof HTMLDataElement;
         HTMLDataListElement: typeof HTMLDataListElement;
         HTMLDetailsElement: typeof HTMLDetailsElement;
-        HTMLDialogElement: typeof HTMLDialogElement;
```

Comment:
Note that JSDOM [actually implements](https://github.com/jsdom/jsdom/blob/master/lib/jsdom/living/nodes/HTMLDialogElement.webidl) this interface, so the type should instead be inlined:
```suggestion
        HTMLDialogElement: {
            new(): HTMLDialogElement;
            readonly prototype: HTMLDialogElement;
        }
```

And the following should be added to the global scope:
```ts
interface HTMLDialogElement {}
```